pkg/eniconfig/eniconfig.go

```diff
@@ -115,10 +116,22 @@ func (h *Handler) Handle(ctx context.Context, event sdk.Event) error {
 				h.controller.myENI = val
 				log.Infof(" Setting myENI to: %s", val)
 			} else {
-				h.controller.eniLock.Lock()
-				defer h.controller.eniLock.Unlock()
-				h.controller.myENI = eniConfigDefault
-				log.Infof(" Setting myENI to: %s", eniConfigDefault)
+
+				label := o.GetLabels()
+
+				val, ok := label[eniConfigLabelDef]
+				if ok {
+					h.controller.eniLock.Lock()
```

Comment:
There are three duplicated `Lock`, `Unlock` lines around this block. Would you like to refactor them?

For example, you can write under L106 as:

```
        case *corev1.Node:

                log.Infof("Handle corev1.Node: %s, %v, %v", o.GetName(), o.GetAnnotations(), o.GetLabels())
                if h.controller.myNodeName == o.GetName() {
                        val, ok := o.GetAnnotations()[eniConfigAnnotationDef]
                        if !ok {
                                val, ok = o.GetLabels()[eniConfigLabelDef]
                                if !ok {
                                        val = eniConfigDefault
                                }
                        }

                        h.controller.eniLock.Lock()
                        defer h.controller.eniLock.Unlock()
                        h.controller.myENI = val
                        log.Infof(" Setting myENI to: %s", val)
                }
        }
        return nil
```